src/css/base/form.scss

```diff
@@ -0,0 +1,23 @@
+
+form {
+  max-width: none;
+}
+
+legend {
+  font-size: 1.2rem;
+  margin-bottom: .5rem;
+}
+
+label {
+  margin-top: 0;
+}
+
+input[type="email"] {
+  font-size: 1rem;
+  padding: .7rem 0.7em;
```

Comment:
use rem for the second one? and specify all four dimensions or explicitly only do `padding-top` and `padding-right`